pkgs/development/web/postman/default.nix

```diff
@@ -0,0 +1,97 @@
+{ stdenv, lib, gnome2, fetchurl, pkgs, xlibs, udev, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "postman-${version}";
+  version = "5.5.2";
+
+  src = fetchurl {
+    url = "https://dl.pstmn.io/download/version/${version}/linux64";
```

Comment:
since you are only fetching the "source" for `x86_64-linux` you should also change `meta.platforms` to `[  "x86_64-linux" ]`.

Reply:
Done